### PR TITLE
WT-7163 Document printlog redacting.

### PR DESCRIPTION
**Pull request by sueloverso**

### CODE REVIEW

File: src/docs/command-line.dox [hunk 1]
```diff
@@ -324,12 +324,13 @@ By default, the \c loadtext command reads from the standard input; the
 
 <hr>
 @section util_printlog wt printlog
-Display the database log.
+Display the database log. By default any operations in the log containing user
+data are redacted.
```
Comment (keithbostic) [inline src/docs/command-line.dox:328]: Is this right? User data is redacted, but the operations themselves are displayed?

File: src/docs/command-line.dox [hunk 2]
```diff
@@ -341,6 +342,9 @@ output; the \c -f option re-directs the output to the specified file.
 @par <code>-m</code>
 Print only message-type log records.
 
+@par <code>-u</code>
+Print user data. Do not redact.
```
Comment (keithbostic) [inline src/docs/command-line.dox:346]: This feels like a weird pair of sentence fragments, maybe just "display user data"?

**Review by keithbostic — APPROVED** [review]

File: src/docs/command-line.dox (hunk 1 above)
Comment (sueloverso) [reply src/docs/command-line.dox:328]: No, the entire operation is redacted if it contains user data. I'm not sure how you're reading that the operation is displayed?

File: src/docs/command-line.dox (hunk 2 above)
Comment (sueloverso) [reply src/docs/command-line.dox:346]: Fixed.

File: src/docs/command-line.dox (hunk 1 above)
Comment (keithbostic) [reply src/docs/command-line.dox:328]: I anticipated the operation itself would be displayed ("transaction XXX changed a row"), but the data itself (the key/value pair), be redacted. Does that make sense?  If that's not what we did, just ignore me!

Comment (sueloverso) [reply src/docs/command-line.dox:328]: That is not what I did. Te entire operation is redacted. It seemed to me that knowing the type and order of operations could leak something even if it wasn't explicitly user data. So I redact the whole thing.

Comment (keithbostic) [reply src/docs/command-line.dox:328]: Yes, I would agree even knowing an operation occurred is a covert channel. Thanks for explaining.

**Pull request merged**